extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/supports/Channels.java

```diff
@@ -54,11 +57,20 @@ public static Channel createChannel(String name) throws SSLException {
             Path keyPath = config.ssl.key.orElse(null);
             SslContextBuilder sslContextBuilder = GrpcSslContexts.forClient();
             if (trustStorePath != null) {
-                sslContextBuilder.trustManager(trustStorePath.toFile());
+                try (InputStream stream = streamFor(trustStorePath, "trust store")) {
+                    sslContextBuilder.trustManager(stream);
+                } catch (IOException e) {
+                    throw new IllegalStateException("Configuring gRPC client trust store failed", e);
```

Comment:
Wondering if you can't use UncheckedIOException here?

Reply:
:+1: it says more than IllegalStateException